docker-compose.yml

```diff
@@ -31,7 +31,10 @@ services:
     restart: ${RESTART_POLICY}
     security_opt:
       - no-new-privileges:true
-    pids_limit: 200
+    pids_limit: 500
+    mem_reservation: 2G
+    mem_limit: 8G
+    memswap_limit: 2G
```

Comment:
I don't think we should enforce these limits by default. I'm fine with documenting them in the README, but a default limit seems too much. There are cases where more than 8G are needed, e.g. for large servers.